src/main/java/com/spring/onlinebookstore/model/Book.java

```diff
@@ -36,4 +45,15 @@ public class Book {
 
     private String description;
     private String coverImage;
+
+    @ManyToMany
+    @JoinTable (
+            name = "books_categories",
+            joinColumns = @JoinColumn(name = "book_id"),
+            inverseJoinColumns = @JoinColumn(name = "category_id")
+    )
+    @Fetch(FetchMode.JOIN)
+    @ToString.Exclude
+    @EqualsAndHashCode.Exclude
```

Comment:
you don't have such methods for book entities. but you can leave it if you want )